src/applications/appeals/10182/containers/IntroductionPage.jsx

```diff
@@ -50,51 +50,53 @@ class IntroductionPage extends React.Component {
         <h2 className="vads-u-font-size--h3">
           Follow the steps below to request a Board Appeal.
         </h2>
+        <h5>
+          If you're requesting a Board Appeal on an issue in a claim we decided
+          before February 19, 2019
+        </h5>
```

Comment:
We should wrap this in a `<strong>` instead of an `<h5>`. For accessibility reasons, we only want to increment headers by one, so if we did make this a header, it should be an `<h3>`; then we add a `vads-u-font-size--h4` to make the size match the design